msteams-platform/tabs/design/tabs.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Design tabs for desktop, web, and mobile
-description: Learn how to design a Teams tab for desktop, web, and mobile, and get the Microsoft Teams UI Kit. Learn about the tab functionality and appearance, building user authentication, tab notifications, and deep linking in tabs.
+description: Learn how to design a tab for desktop, web, and mobile, and get the Microsoft Teams UI Kit. Learn about the tab, building user authentication, tab notifications, and deep linking.
```

Comment:
The first sentence reads something like, Learn how to get UI kit. Needs an edit.

Reply:
@ashishguptaiitb,

Edited the metadata description in this PR: https://github.com/MicrosoftDocs/msteams-docs/pull/6385